test/e2e-tests/config.json

```diff
@@ -0,0 +1,34 @@
+{
+  "NAMESPACE": "",
+  "IMAGE_REGISTRY_USERNAME": "",
+  "IMAGE_REGISTRY_PASSWORD": "",
+  "TARGET_IMAGE_TAGS": [],
+
```

Comment:
I created a template.config.json which can be copied and have its values filled in. The copy needs to be called config.json for the e2e-tests, and I added the config.json to .gitignore to prevent leaks.

So `cp template.config.json config.json` -> fill in fields -> and git isn't tracking you actual config

Reply:
* Also fixed the issue you mentioned